meadow-client/src/Types.purs

```diff
@@ -115,154 +64,117 @@ type ChildSlot
 
-_createGistResult ::
-  forall s a.
-  Lens' {createGistResult :: a | s} a
+_createGistResult :: forall s a. Lens' {createGistResult :: a | s} a
 _createGistResult = prop (SProxy :: SProxy "createGistResult")
 
-_marloweState ::
-  forall s a.
-  Lens' {marloweState :: a | s} a
+_marloweState :: forall s a. Lens' {marloweState :: a | s} a
 _marloweState = prop (SProxy :: SProxy "marloweState")
 
-
 -- Oracles should not be grouped (only one line per oracle) like:
 --    Oracle 3: Provide value [$value] for block [$timestamp]
-type OracleEntry = { timestamp :: BlockNumber -- editable
-                   , value :: BigInt }        -- editable
+type OracleEntry
+  = {timestamp :: BlockNumber, value :: BigInt}
 
+-- editable
 _timestamp ::
   forall s a.
   Lens' {timestamp :: a | s} a
 _timestamp = prop (SProxy :: SProxy "timestamp")
 
-_value ::
-  forall s a.
-  Lens' {value :: a | s} a
+_value :: forall s a. Lens' {value :: a | s} a
 _value = prop (SProxy :: SProxy "value")
 
-type InputData = { oracleData :: Map IdOracle OracleEntry -- oracle inputs (before person
-                 , inputs :: Map Person PersonInput } -- inputs (grouped by Person id)
+type InputData
+  = {oracleData :: Map IdOracle OracleEntry, inputs :: Map Person PersonInput}
 
+-- inputs (grouped by Person id)
 _oracleData ::
   forall s a.
   Lens' {oracleData :: a | s} a
 _oracleData = prop (SProxy :: SProxy "oracleData")
 
-_inputs ::
-  forall s a.
-  Lens' {inputs :: a | s} a
+_inputs :: forall s a. Lens' {inputs :: a | s} a
 _inputs = prop (SProxy :: SProxy "inputs")
 
-type TransactionData = { inputs :: Array (Either (Tuple Person PersonInput) (Tuple IdOracle OracleEntry))
-                                  -- ^ not grouped (but participants and oracles labelled inline)
-                                  -- e.g: Participant 1 - Action $IdAction - Commit ....
-                                  --      Oracle 3 - Provide value $value for block $timestamp
-                       , signatures :: Map Person Boolean -- checkboxes
-                       , outcomes :: Map Person BigInt } -- table under checkboxes
+type TransactionData
+  = {inputs :: Array (Either (Tuple Person PersonInput) (Tuple IdOracle OracleEntry)), signatures :: Map Person Boolean, outcomes :: Map Person BigInt}
 
+-- table under checkboxes
 _signatures ::
   forall s a.
   Lens' {signatures :: a | s} a
 _signatures = prop (SProxy :: SProxy "signatures")
 
-_outcomes ::
-  forall s a.
-  Lens' {outcomes :: a | s} a
+_outcomes :: forall s a. Lens' {outcomes :: a | s} a
 _outcomes = prop (SProxy :: SProxy "outcomes")
 
-data PersonInput =
-   CommitAction IdAction IdCommit Value Timeout -- "Action $IdAction: Commit $Value ADA with id $IdCommit until block $Timeout"
- | PayClaim IdAction IdCommit Value -- "Action $IdAction: Claim a payment of $Value ADA from commit $IdCommit"
- | ProvideChoice IdChoice Choice -- "Choice $IdChoice: Choose value [$Choice]"
+data PersonInput
```

Comment:
Same question - docs gone deliberately?

Reply:
nope and I'm not sure what to do, TBH I want to argue for using purty despite all its problems.